### ISSUE TITLE
Include SHA ID in version string

### ISSUE THREAD

**Issue by budhrg** [minishift/minishift#515]
I am thinking here that should we include SHA ID in the version command?

I observed that both Openshift origin and Docker machine includes SHA ID in their version as

```
$ oc version
oc v1.4.0+208f053
...

$ docker-machine version
docker-machine version 0.10.0, build 76ed2a6
```

I am thinking of this for minishift :
```
$ minishift version
Minishift version: 1.0.0-beta.4, build 8102f95
```

Any thoughts?

**Comment by hferentschik** [comment]
I am basically ok with this, but don't we use the version as well for checks against GitHub? We cannot just append the sha I would think, but rather pass it in separately and then just append it when `minishift version` is called. Right?

**Comment by budhrg** [comment]
> We cannot just append the sha I would think, but rather pass it in separately and then just append it when minishift version is called. Right?

Yes, similar to how minishift version / openshift version are being used presently.

**Comment by hferentschik** [comment]
Merged via pull request #1094
